### PR TITLE
Added some examples

### PR DESCRIPTION
**Pull request by gpenverne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,39 @@ Here is a sample response from the GDC api :
...
+## Example
+
+### Ask permission to user
+https://gensdeconfiance.fr/oauth/v2/auth?client_id=``your-client_id``&response_type=code&redirect_uri=``your-redirect-uri``
+This will redirect user to your callback url with a ``code`` parameter
+
+Example:
+https://gensdeconfiance.fr/oauth/v2/auth?client_id=1_23ABCDE&response_type=code&redirect_uri=http%3A%2F%2Fgdc.local%2Ftest
+Will redirect to:
+https://gensdeconfiance.fr/test?code=abcde1234
```
Comment (ndavoust) [inline README.md]: insconsistent with the redirect_uri param :)
use https://exemple.com/oauth

Comment (ndavoust) [reply README.md]: is it "code" or "token" ?

File: README.md [hunk 2]
```diff
@@ -53,3 +53,39 @@ Here is a sample response from the GDC api :
...
+}
+```
+Or if error:
+```json
+{
+    "error":"invalid_grant",
+    "error_description":"The authorization code has expired"
+}
+```
+
```
Comment (ndavoust) [inline README.md]: and then the last step ? how to use the access_token to retreive user data ?

File: README.md (hunk 1 above)
Comment (gpenverne) [reply README.md]: code. It is an authorization code used to retrieve a token

File: README.md (hunk 2 above)
Comment (gpenverne) [reply README.md]: In another PR ;)

**Comment by kbsali** [comment]
to be continued

**Pull request merged**